Reduce size of the btm binary from 3.8 -> 2.6 MiB (in version 0.6.3).

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -27,8 +27,9 @@
 lto = true
 # debug = true
 # lto = false
-opt-level = 3
+opt-level = "z"
 codegen-units = 1
+panic = "abort"
 
 [features]
 default = ["fern", "log"]
